packages/runtime/src/schema.ts

```diff
@@ -178,23 +178,12 @@ const UpdateOptionsInput = new GraphQLInputObjectType({
   },
 })
 
-const CeramicSignerInputType = new GraphQLInputObjectType({
-  name: 'CeramicSignerInput',
-  fields: {
-    isAuthenticated: { type: GraphQLBoolean },
-  },
-})
-
 const HideOptionsInput = new GraphQLInputObjectType({
```

Comment:
Could you rename the input as we discussed please?
```suggestion
const EnableIndexingOptionsInput = new GraphQLInputObjectType({
```

Reply:
I had forgotten this, thanks for the reminder. Ready!